packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
FYI @TimRoe, Had to add this because when I tried testing in the app, I would get the following TS error:
```
node_modules/@department-of-veterans-affairs/mobile-component-library/src/components/Snackbar/Snackbar.tsx(57,34): error TS7017: Element implicitly has an 'any' type because type 'typeof globalThis' has no index signature. 
error Command failed with exit code 2.
```

Reply:
Hmmm, that's fine for this ticket but we should figure out something better long term where one doesn't need the config in global.d.ts to not get errors when not even using the component. Perhaps create a ticket just for that nearer the end of Snackbar implementation to figure out the least invasive way to go about it.

Reply:
Yeah, I was implying that this is a temporary fix and assume we'd fix it as part of the implementation.

Reply:
Created ticket to make sure we don't miss this: https://github.com/department-of-veterans-affairs/va-mobile-library/issues/412